modules/network-observability-operator-release-notes-1-10-new-features-enhancements.adoc

```diff
@@ -0,0 +1,48 @@
+// Module included in the following assemblies:
+// * network_observability/network-observability-release-notes-1-10.adoc
+
+:_mod-docs-content-type: REFERENCE
+[id="network-observability-operator-release-notes-1-10-new-features-enhancements_{context}"]
+= Network Observability Operator 1.10 new features and enhancements
+
+[role="_abstract"]
+The Network Observability Operator 1.10 release enhances security, improves performance, and introduces new CLI/UI tools for better network flow management.
+
+[id="network-policy-updates_{context}"]
+== Network policy updates
+The Network Observability Operator now supports configuring both ingress and egress network policies to control pod traffic. This enhancement improves security and ensures you collect only the network flow data you need.
```

Comment:
not sure what is meant exactly by " and ensures you collect only the network flow data you need"
I'd remove that?